x-pack/plugins/security_solution/public/cases/containers/use_get_fields.tsx

```diff
@@ -1,82 +0,0 @@
-/*
```

Comment:
I used this book in the flyout I had while I was working on my demo UI, but I am actually not going to expose the fields and instead have a call for the mappings. we will not need this hook